symengine/tests/polynomial/test_uratpoly_flint.cpp

```diff
@@ -3,7 +3,9 @@
 
 #include <symengine/polys/uintpoly_flint.h>
 #include <symengine/pow.h>
+#include <symengine/symengine_exception.h>
 
+using SymEngine::SymEngineException;
```

Comment:
`SymEngineException` can be renamed to just `Exception` since we are using it inside `SymEngine` namespace.